main.go

```diff
@@ -18,7 +18,8 @@ import (
 )
 
 var (
-	deck *Deck
+	deck        *Deck
+	currentDeck string
```

Comment:
We could probably use `deck.File` here, instead of another global variable.

Reply:
Makes sense, done.